tests/utils/next-version-helpers.mjs

```diff
@@ -85,7 +85,7 @@ export async function setNextVersionInFixture(
         if (
           operation === 'update' &&
           versionConstraint &&
-          !satisfies(checkVersion, versionConstraint) &&
+          !satisfies(checkVersion, versionConstraint, { includePrerelease: true }) &&
```

Comment:
not needed for this PR, but I noticed that `after` fixture building was skipped in canary test run which should not happen (we don't have integration test using that fixture, so it wasn't impacting test results - this is just fixing this for the future)